From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Wed, 28 Jan 2015 15:02:54 +0100
Subject: ar71xx: add default LED configuration for D-Link DIR-615 rev. C1

This patch adds an entry in the uci-defaults' led-file to configure the
WAN and WLAN LEDs by default.

Signed-off-by: Vincent Wiemann <me@bibbl.com>

diff --git a/target/linux/ar71xx/base-files/etc/uci-defaults/01_leds b/target/linux/ar71xx/base-files/etc/uci-defaults/01_leds
index 2676b48..ee90c52 100755
--- a/target/linux/ar71xx/base-files/etc/uci-defaults/01_leds
+++ b/target/linux/ar71xx/base-files/etc/uci-defaults/01_leds
@@ -122,6 +122,11 @@ dir-615-e4)
 	ucidef_set_led_switch "lan4" "LAN4" "d-link:green:lan4" "switch0" "0x10"
 	;;
 
+dir-615-c1)
+	ucidef_set_led_netdev "wan" "WAN" "d-link:green:wan" "eth1"
+	ucidef_set_led_wlan "wlan" "WLAN" "d-link:green:wlan" "phy0tpt"
+	;;
+
 dir-825-b1)
 	ucidef_set_led_usbdev "usb" "USB" "d-link:blue:usb" "1-1"
 	;;
